src/vos/tests/evt_ctl.sh

```diff
@@ -10,8 +10,8 @@ DAOS_DIR=$(cd "${cwd}/../../.." && echo "$PWD")
 source "$DAOS_DIR/.build_vars.sh"
 EVT_CTL="$SL_PREFIX/bin/evt_ctl"
 
-cmd_create="$VCMD $EVT_CTL $* -C o:4"
-cmd=$cmd_create
+cmd="$VCMD $EVT_CTL $* --start-test \"EVT030: internal tests and evt sequence\"\
```

Comment:
need to remove the \" here.   The test name that gets inserted is EVT030:

Also, I think we should make this evt sequence and then run a separate command for evt internal tests like we do with evt_drain